src/mbgl/renderer/image_manager_observer.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+namespace mbgl {
+
+class ImageManagerObserver {
+public:
+    virtual ~ImageManagerObserver() = default;
+
+    virtual void onStyleImageMissing(const std::string&, std::function<void()>) {}
```

Comment:
nit: would it be better to define alias for `std::function<void()>`, easier to change in the future and might improve readability?

Reply:
done in 6089031